include/sound/soc-dapm.h

```diff
@@ -604,6 +604,7 @@ struct snd_soc_dapm_widget {
 	unsigned char power_checked:1;		/* power checked this run */
 	unsigned char is_supply:1;		/* Widget is a supply type widget */
 	unsigned char is_ep:2;			/* Widget is a endpoint type widget */
+	unsigned char no_wname_in_long_name:1;	/* No widget name prefix in kcontrol name */
```

Comment:
commit message: the last sentence is missing something, the grammar does not work.

this flag gives the driver an option to drop it if the name
from the snd_kcontrol_new is meaningful enough.